### ISSUE TITLE
improve Report class

### ISSUE THREAD

**Issue by kojisekig** [LTR4L/ltr4l#5]
Report class has some problems,

* it closes data.csv file every line
* Main class doesn't allow uses to change the file name
* README.md in docs folder regards data folder as the folder for dataset (training set and validation set). The (default) file and folder names should be changed, such as report.csv, result.csv etc.
